<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_top</class>
 <widget class="QMainWindow" name="main_top">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="main_window">
      <item>
       <widget class="QTabWidget" name="grade_tab">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_1">
         <property name="toolTip">
          <string/>
         </property>
         <attribute name="title">
          <string>整数算术题</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="input_range">
            <property name="title">
             <string>出题范围</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="input_range_min" stretch="1,3">
               <item>
                <widget class="QLabel" name="input_range_min_text">
                 <property name="text">
                  <string>起始数字：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="input_range_min_num">
                 <property name="text">
                  <string>1</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="input_range_max" stretch="1,3">
               <item>
                <widget class="QLabel" name="input_range_max_text">
                 <property name="text">
                  <string>终止数字：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="input_range_max_num">
                 <property name="text">
                  <string>10</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="input_options">
            <property name="title">
             <string>出题参数</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="checkbox_tf_line1" stretch="1,1,1">
               <item>
                <widget class="QCheckBox" name="checkbox_tf_plus">
                 <property name="text">
                  <string>加法运算</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkbox_tf_minus">
                 <property name="text">
                  <string>减法运算</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkbox_tf_compare">
                 <property name="text">
                  <string>比较运算</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="checkbox_tf_line2" stretch="1,1,1">
               <item>
                <widget class="QCheckBox" name="checkbox_tf_multi">
                 <property name="text">
                  <string>乘法运算</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkbox_tf_division">
                 <property name="text">
                  <string>除法运算</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkbox_tf_remainder">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>余数运算</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="input_question_number">
            <property name="title">
             <string>出题数量</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QHBoxLayout" name="input_questions" stretch="1,3">
               <item>
                <widget class="QLabel" name="input_questions_text">
                 <property name="text">
                  <string>出题数量：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="input_questions_num">
                 <property name="text">
                  <string>10</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_generate_1">
            <property name="text">
             <string>生成题目</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="output_text">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>180</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::LineWrapMode::WidgetWidth</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
